File: android/src/main/kotlin/au/com/annon/flutter_mapbox_turn_by_turn/models/MapboxLocationChangeEvent.kt
package au.com.annon.flutter_mapbox_turn_by_turn.models

class MapboxLocationChangeEvent(location: com.mapbox.common.location.Location) {
    val isLocationChangeEvent = true
    var latitude: Double? = null
    var longitude: Double? = null

    init {
        latitude = location.latitude
        longitude = location.longitude
    }
}

File: ios/Classes/models/MapboxLocationChangeEvent.swift
import Foundation

public class MapboxLocationChangeEvent: Codable {
  let isLocationChangeEvent: Bool
  let latitude: Double?
  let longitude: Double?

  init(latitude: Double?, longitude: Double?) {
    self.isLocationChangeEvent = true
    self.latitude = latitude
    self.longitude = longitude
  }
}
